manifests/params.pp

```diff
@@ -23,13 +23,22 @@
       $tmp_dir         = '/tmp'
       $install_dir     = undef
       $download_url    = undef
+      case $::osfamily {
+        'RedHat': {
+          $service_provider = 'redhat'
+        }
```

Comment:
What happens if `service_provider` is set to redhat on versions < 7? Does everything still work?